fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2-datagen/lang/en_us.json

```diff
@@ -0,0 +1,300 @@
+  "tag.item.c.dusts.redstone": "Redstone Dusts",
+  "tag.item.c.dyed": "Dyed Items",
+  "tag.item.c.dyed.black": "Black Dyed Items",
+  "tag.item.c.dyed.blue": "Blue Dyed Items",
+  "tag.item.c.dyed.brown": "Brown Dyed Items",
+  "tag.item.c.dyed.cyan": "Cyan Dyed Items",
+  "tag.item.c.dyed.gray": "Gray Dyed Items",
+  "tag.item.c.dyed.green": "Green Dyed Items",
+  "tag.item.c.dyed.light_blue": "Light Blue Dyed Items",
+  "tag.item.c.dyed.light_gray": "Light Gray Dyed Items",
+  "tag.item.c.dyed.lime": "Lime Dyed Items",
+  "tag.item.c.dyed.magenta": "Magenta Dyed Items",
+  "tag.item.c.dyed.orange": "Orange Dyed Items",
+  "tag.item.c.dyed.pink": "Pink Dyed Items",
+  "tag.item.c.dyed.purple": "Purple Dyed Items",
+  "tag.item.c.dyed.red": "Red Dyed Items",
+  "tag.item.c.dyed.white": "White Dyed Items",
+  "tag.item.c.dyed.yellow": "Yellow Dyed Items",
+  "tag.item.c.dyes": "Dyes",
+  "tag.item.c.dyes.black": "Black Dyes",
+  "tag.item.c.dyes.blue": "Blue Dyes",
+  "tag.item.c.dyes.brown": "Brown Dyes",
+  "tag.item.c.dyes.cyan": "Cyan Dyes",
+  "tag.item.c.dyes.gray": "Gray Dyes",
+  "tag.item.c.dyes.green": "Green Dyes",
+  "tag.item.c.dyes.light_blue": "Light Blue Dyes",
+  "tag.item.c.dyes.light_gray": "Light Gray Dyes",
+  "tag.item.c.dyes.lime": "Lime Dyes",
+  "tag.item.c.dyes.magenta": "Magenta Dyes",
+  "tag.item.c.dyes.orange": "Orange Dyes",
+  "tag.item.c.dyes.pink": "Pink Dyes",
+  "tag.item.c.dyes.purple": "Purple Dyes",
+  "tag.item.c.dyes.red": "Red Dyes",
+  "tag.item.c.dyes.white": "White Dyes",
+  "tag.item.c.dyes.yellow": "Yellow Dyes",
+  "tag.item.c.enchantables": "Enchantables",
+  "tag.item.c.foods": "Foods",
+  "tag.item.c.foods.berries": "Berries",
+  "tag.item.c.foods.breads": "Breads",
+  "tag.item.c.foods.candies": "Candies",
+  "tag.item.c.foods.cooked_fishes": "Cooked Fishes",
+  "tag.item.c.foods.cooked_meats": "Cooked Meats",
+  "tag.item.c.foods.cookies": "Cookies",
+  "tag.item.c.foods.edible_when_placed": "Edible When Placed",
+  "tag.item.c.foods.food_poisoning": "Food Poisoning",
+  "tag.item.c.foods.fruits": "Fruits",
+  "tag.item.c.foods.raw_fishes": "Raw Fishes",
+  "tag.item.c.foods.raw_meats": "Raw Meats",
+  "tag.item.c.foods.soups": "Soups",
+  "tag.item.c.foods.vegetables": "Vegetables",
+  "tag.item.c.gems": "Gems",
+  "tag.item.c.gems.amethyst": "Amethyst Gems",
+  "tag.item.c.gems.diamond": "Diamond Gems",
+  "tag.item.c.gems.emerald": "Emerald Gems",
+  "tag.item.c.gems.lapis": "Lapis Gems",
+  "tag.item.c.gems.prismarine": "Prismarine Gems",
+  "tag.item.c.gems.quartz": "Quartz Gems",
+  "tag.item.c.glass_blocks": "Glass Blocks",
+  "tag.item.c.glass_panes": "Glass Panes",
+  "tag.item.c.hidden_from_recipe_viewers": "Hidden From Recipe Viewers",
+  "tag.item.c.ingots": "Ingots",
+  "tag.item.c.ingots.copper": "Ingots Copper",
+  "tag.item.c.ingots.gold": "Ingots Gold",
+  "tag.item.c.ingots.iron": "Ingots Iron",
+  "tag.item.c.ingots.netherite": "Ingots Netherite",
```

Comment:
Should be `Copper Ingots`, `Gold Ingots`, `Iron Ingots` and `Netherite Ingots`

Reply:
Corrected now